table/server/master/src/main/java/alluxio/master/table/CatalogProperty.java

```diff
@@ -38,4 +38,7 @@ private CatalogProperty(String name, String description, String defaultValue) {
               + "overload the UDB, and if set too low, syncing a database with many tables may "
               + "take a long time.",
           Integer.toString(DEFAULT_DB_SYNC_THREADS));
+  public static final CatalogProperty DB_CONFIG_FILE =
+      new CatalogProperty("catalog.db.config.file",
+          "The config file for the UDB.", "<catalog.db.config.file>");
```

Comment:
It seems there is only one add on configuration for by pass, it will be better to use single configuration with helper function. Check `alluxio.conf.InstancedConfiguration::getList` for reference.

Reply:
I'm afraid that I have to keep using `config.file` since I want to refresh the bypass list by `alluxio table sync` command, further, I want to add more config item in this file.

Reply:
I see, that make sense to me.